cloudflare_worker/worker/src/streams.test.ts

```diff
@@ -32,12 +32,15 @@ describe('readIntoArray', () => {
   it('reads a stream with two chunks', async () => {
     const {writable, readable} = new TransformStream;
     const writer = writable.getWriter();
-    for (let i = 0; i < 2; i++) {
-      writer.write(new TextEncoder().encode('hello'));
-    }
-    writer.close();
+    const write = (async() => {
```

Comment:
(Optional) Declare `const writer` variable inside this `async() => {...}` closure.